run_create_sqs.py

```diff
@@ -74,6 +74,71 @@ def run_create_queue(name, settings):
     aws_cli.run(cmd)
 
 
+def run_create_fifo_queue(name, settings):
+    print_message(f'create sqs queue: {name}')
+
+    delay_seconds = settings['DELAY_SECONDS']
+    receive_count = settings['RECEIVE_COUNT']
+    receive_message_wait_time_seconds = settings['RECEIVE_MESSAGE_WAIT_TIME_SECONDS']
+    retention = settings['RETENTION']
+    timeout = settings['TIMEOUT']
+    use_redrive_policy = settings['USE_REDRIVE_POLICY']
+    dead_letter_queue_arn = None
+
+    aws_cli = AWSCli(settings['AWS_REGION'])
+
+    if use_redrive_policy == "True":
+        attributes = dict()
+        attributes['MessageRetentionPeriod'] = retention
+        attributes['FifoQueue'] = "true"
+        attributes['ContentBasedDeduplication'] = "true"
+
+        queue_name = name.replace('.fifo', '')
+
+        cmd = ['sqs', 'create-queue']
+        cmd += ['--queue-name', f'{queue_name}-dead-letter.fifo']
+        cmd += ['--attributes', json.dumps(attributes)]
+        aws_cli.run(cmd)
+
+        elapsed_time = 0
+        while True:
+            cmd = ['sqs', 'get-queue-url', '--queue-name', f'{queue_name}-dead-letter.fifo']
+            result = aws_cli.run(cmd)
+
+            if type(result) == dict:
+                if result.get('QueueUrl', None):
+                    break
```

Comment:
@GoldieKMG // depth를 줄이는 것을 제안하여 드립니다 🙏 
```suggestion
            if type(result) == dict and result.get('QueueUrl', None):
                break
```